.env

```diff
@@ -0,0 +1 @@
+RESEND_API_KEY=re_Xf5Kxt5W_JUkMJkujRDSvXKd8Hs6uxSkH
```

Comment:
Pamiętaj, aby kluczy nie wrzucać na GitHuba, bo mogą zostać łatwo przejęte. Jeśli chcesz mi wysłać taki kod to zrób to na slacku :)